src/sync/sync_user.hpp

```diff
@@ -64,13 +64,18 @@ friend class SyncSession;
     // Log the user out and mark it as such. This will also close its associated Sessions.
     void log_out();
 
-    // Whether the user was configured as an 'admin user' (directly uses its user token
-    // to open Realms).
+    // Whether the user has administrator privileges.
     bool is_admin() const
     {
         return m_is_admin;
     }
 
+    // Specify whether the user has administrator privileges.
+    // Note that this is an internal flag meant for bindings to communicate information
+    // originating from the server. It is *NOT* possible to unilaterally change a user's
+    // administrator status from the client through this or any other API.
+    void update_admin_status(bool);
```

Comment:
I had originally wanted to make it clear that changing the bool wouldn't actually change the user's admin status on the server, but I think that concern is overkill.